packages/ckeditor5-image/src/image/imageediting.js

```diff
@@ -98,17 +98,12 @@ export default class ImageEditing extends Plugin {
 
 		conversion.for( 'upcast' )
 			.elementToElement( {
-				view: matchImageInsideParent( 'p' ),
-				model: ( viewImage, { writer } ) => writer.createElement( 'imageInline', { src: viewImage.getAttribute( 'src' ) } )
+				view: matchImageByType( false ),
```

Comment:
I would replace `true`/`false` with `'inline'`/`'block'` to make this code clear without checking this function signature.

Reply:
I agree. Alternatively:

*   you could use `'image'` and `'imageInline'` strings which are everywhere anyway,
*   there could be `const IMAGE_TYPE_INLINE = 1` and `const IMAGE_TYPE_BLOCK = 2` declared at the top of the file and used here (or anywhere else),

Reply:
This notation should also be shared with `createImageViewElement()`.